src/components/Zara.css

```diff
@@ -1,6 +1,9 @@
 .zara-wrapper {
   background-color: rgb(158, 193, 213);
-  padding: 250px 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
So i replaced your `padding: 250px 0` with the 4 lines of code above.

By using 250px of y-axis padding, it assumes that the height of the viewport across user screens will be the same, which is not the case. Because of this, we want the Zara Rahim page to take up the whole viewport size regardless of the users screen. Therefore we set the height to `100vh`, which is 100% of the viewport height.

Next, by doing that, we lose the vertical center placement of the Zara Rahim `h1`. We display `flex` the container and `justify-content:center` to center it horizontally on the screen, and then `align-items:center` to center it vertically on the screen.